### PR TITLE
#161962033 Fix checkbox and multi-trip cancel icon designs in request modal

### PR DESCRIPTION
**Pull request by ezrqnkemboi**
#### What does this PR do?

- Fix checkbox and multi-trip cancel icon designs in request modal
#### Description of Task to be completed?
- fix checkbox request modal to match mockup
- fix cancel icon in multi-trip request to match mockup

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- git checkout bg-fix-request-modal-design-161962033
- yarn start
- click new request
- check the alignment of the send request 
- check the cancel icon in multi-trip
- check the mockups

#### Any background context you want to provide?
[Request Form Mockup](https://projects.invisionapp.com/d/#/console/14861108/319652031/preview)
[Multi-trip cancel icon mockup](https://projects.invisionapp.com/d/#/console/14861108/319652029/preview)

#### What are the relevant pivotal tracker stories?
[#161962033](https://www.pivotaltracker.com/story/show/161962033)

#### Screenshots (if appropriate)
screen shot 2018-11-15 at 00 46 25

screen shot 2018-11-15 at 20 38 46

#### Questions:

### CODE REVIEW

**Comment by ronaldndirangu** [comment]
Kindly resolve the conflicts @ezrqnkemboi

**Comment by ezrqnkemboi** [comment]
@ronaldndirangu feedback is received and working on those. I had not checked on that cancel. It is a good catch up.

**Comment by ezrqnkemboi** [comment]
@ronaldndirangu Actually I have noticed that the mockup is Cancel while we have put it Clear. When you look keenly, the button clears data in the fields. That means that if we change it to cancel as in mockup, it will confuse the end-user. I prefer it to remain with the name Clear.

File: src/components/Forms/NewAccommodationForm/__tests__/__snapshots__/NewAccommodation.tests.js.snap [hunk 1]
```diff
@@ -415,7 +415,7 @@ exports[`<NewAccommodation /> renders correctly 1`] = `
   <hr />
   <fieldset>
     <div
-      className="submit-area"
+      className="submit-area submit-area--undefined"
```
Comment (DennisWanjiru) [inline src/components/Forms/NewAccommodationForm/__tests__/__snapshots__/NewAccommodation.tests.js.snap]: why is does this class have undefined. Check how you are passing your props and how you are validating.

File: src/components/Forms/NewUserRoleForm/__tests__/__snapshots__/NewUserRoleForm.test.js.snap [hunk 2]
```diff
@@ -98,7 +98,7 @@ exports[`<NewUserRoleForm /> renders correctly 1`] = `
   <hr />
   <fieldset>
     <div
-      className="submit-area"
+      className="submit-area submit-area--undefined"
```
Comment (DennisWanjiru) [inline src/components/Forms/NewUserRoleForm/__tests__/__snapshots__/NewUserRoleForm.test.js.snap]: Please also check here, make the necessary changes.

File: src/components/Forms/NewAccommodationForm/__tests__/__snapshots__/NewAccommodation.tests.js.snap (hunk 1 above)
Comment (ezrqnkemboi) [reply src/components/Forms/NewAccommodationForm/__tests__/__snapshots__/NewAccommodation.tests.js.snap]: I have realized that the Submit Button is reused in these forms. Let me try to fix this issue.

File: src/components/Forms/NewUserRoleForm/__tests__/__snapshots__/NewUserRoleForm.test.js.snap (hunk 2 above)
Comment (ezrqnkemboi) [reply src/components/Forms/NewUserRoleForm/__tests__/__snapshots__/NewUserRoleForm.test.js.snap]: Checking on this also.

**Comment by amarachukwu-agbo** [comment]
@ezrqnkemboi, the cancel icon looks aligned, however, the text beside the checkbox is still not vertically aligned with the checkbox. Please look into this.
Also, maintain consistency with the case you are using to start your sentences in the PR description

**Comment by ezrqnkemboi** [comment]
@amarachukwu-agbo feedback received and being worked on. I hope it is the alignment for multi-trip.

**Comment by samuelkubai** [comment]
@ezrqnkemboi kindly resolve the conflicts.

**Comment by ezrqnkemboi** [comment]
Working on that @samuelkubai

**Comment by samuelkubai** [comment]
Cool @ezrqnkemboi.

**Pull request merged**